techniques/failures/F97.html

```diff
@@ -2,98 +2,81 @@
+      <h1>達成基準 1.3.4 の失敗例 － 表示の向きを縦向き (portrait) 又は横向き (landscape) に固定している</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/orientation">Success Criterion 1.3.4: Orientation</a> (Failure)</span>.
+            <h2>適用 (対象)</h2>
+            <p>この達成方法は、<span><a href="https://waic.jp/docs/WCAG21/Understanding/orientation">達成基準 1.3.4: 表示の向き</a> (失敗)</span> に関連する。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to describe how restricting the view of content
-               to a single orientation is a failure to allow content to be viewed in multiple orientations.
-               When content is presented with a restriction to a specific orientation, users must
-               orient their devices to view the content in the orientation that the author imposed.
-               Some users have their devices mounted in a fixed orientation (e.g. on the arm of a
-               power wheelchair), and if the content cannot be viewed in that orientation it creates
-               problems for the user.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、コンテンツの表示を単一の向きに制限することが、コンテンツを複数の向きで表示することの妨げになることを記述することである。コンテンツが特定の向きに制限されて提示されている場合、利用者は、コンテンツ制作者が課した向きでコンテンツを表示するようにデバイスを向けなければならない。利用者の中には、デバイスを固定された向き (例えば、電動車椅子のアームの上など) で取り付けている人もいて、その向きでコンテンツを見ることができない場合、利用者にとって問題が生じてしまう。
             </p>
-            <p>If a specific orientation is determined to be <a href="https://w3c.github.io/wcag/guidelines/#" target="terms">essential</a> for the operation and viewing of the content, then this failure technique will not
-               apply.
+            <p>特定の向きが、コンテンツを表示するうえで<a href="https://waic.jp/docs/WCAG21/#dfn-essential">必要不可欠</a>と判断される場合は、この達成方法 (失敗例) は適用されない。
```

Comment:
operationがどこかに行ってしまったので
```suggestion
            <p>特定の向きが、コンテンツの操作及び表示に<a href="https://waic.jp/docs/WCAG21/#dfn-essential">必要不可欠</a>と判断される場合は、この達成方法 (失敗例) は適用されない。
```